easybuild/easyconfigs/p/pkg-config/pkg-config-0.29.2-GCCcore-11.3.0.eb

```diff
@@ -0,0 +1,34 @@
+easyblock = 'ConfigureMake'
+
+name = 'pkg-config'
```

Comment:
@jfgrimm Since we intend to use `pkgconf` rather than `pkg-config`, we should not add an easyconfig file for `pkg-config` with `GCCcore/11.3.0`?

Reply:
I've removed the `pkg-config` easyconfig file with `GCCcore/11.3.0` in 3db546c so we can move this forward...